dev.env

```diff
@@ -0,0 +1,29 @@
+PRINT_ENV=dev
+RANCHER_LABEL=dev
+IMAGE_TAG=staging
+NGINX_PORT=8009
+TOMCAT_PORT=8011
+API_URL=//mf-chsdi3.dev.bgdi.ch
+TOMCAT_SERVER_URL=//vpc-alb-bgdi-dev.intra.bgdi.ch
+PRINT_SERVER_HOST=service-print.dev.bgdi.ch
+PRINT_TEMP_DIR=/var/local/print
+PRINT_LOGLEVEL=10
+DEBUG=True
+APACHE_ENTRY_PATH=
+APACHE_BASE_PATH=main
+BRANCH_STAGING=
+GIT_BRANCH=
+MODWSGI_USER=www-data
+INSTALLDIR=/var/www/vhosts/mf-chsdi3/private/chsdi
+WSGI_APP=/var/www/vhosts/mf-chsdi3/private/chsdi/apache/application.wsgi
+KML_TEMP_DIR=/var/local/print/kml
+DEPLOY_TARGET=prod
+MODWSGI_CONFIG=production.ini
+ROBOTS_FILE=robots_prod.txt
```

Comment:
is there a robots_dev.txt?

Reply:
Well, _jein_, it's actually called `robots.txt` (don't ask why)